Build/Projects/MonoGame.Framework.definition

```diff
@@ -1289,6 +1155,423 @@
       <Services>DirectXGraphics</Services>
     </EmbeddedResource>
 
+		<!-- PCL Platform -->
```

Comment:
If this set of files is shared between the PCL project and Linux/Web, you'll be better off using an Include Project: https://protobuild.readthedocs.org/en/latest/include_projects.html  Put all of the files (with no platform conditionals) in the include project, then reference the include project from the PCL, and reference the include project under the platform conditionals for the FrameworkReferences external project.